Lack of dependencies for gazebo_ros_gps

Hi,

When I pulled `melodic-devel` (commit: 7d5e26d51187d12924a87947f230e7fe38e0206e) , `catkin build` is failed after `catkin clean`.
I think the `hector_gazebo_ros_gps` need  `${PROJECT_NAME}_generate_messages_cpp` in add_dependencies.

I'll send PR, Please confirm.

Error message:
```
Errors     << hector_gazebo_plugins:make /home/gisen/ros/logs/hector_gazebo_plugins/build.make.000.log
In file included from /home/gisen/ros/src/hector-gazebo/hector_gazebo_plugins/src/gazebo_ros_gps.cpp:29:0:
/home/gisen/ros/src/hector-gazebo/hector_gazebo_plugins/include/hector_gazebo_plugins/gazebo_ros_gps.h:37:10: fatal error: hector_gazebo_plugins/SetReferenceGeoPose.h: No such file or directory
 #include <hector_gazebo_plugins/SetReferenceGeoPose.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hector_gazebo_ros_gps.dir/src/gazebo_ros_gps.cpp.o] Error 1
make[1]: *** [CMakeFiles/hector_gazebo_ros_gps.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make: *** [all] Error 2
```

--- a/hector_gazebo_plugins/CMakeLists.txt
+++ b/hector_gazebo_plugins/CMakeLists.txt
@@ -88,7 +88,7 @@ add_dependencies(hector_gazebo_ros_magnetic ${PROJECT_NAME}_gencfg)
 
 add_library(hector_gazebo_ros_gps src/gazebo_ros_gps.cpp)
 target_link_libraries(hector_gazebo_ros_gps ${GAZEBO_LIBRARIES} ${catkin_LIBRARIES})
-add_dependencies(hector_gazebo_ros_gps ${PROJECT_NAME}_gencfg)
+add_dependencies(hector_gazebo_ros_gps ${PROJECT_NAME}_generate_messages_cpp ${PROJECT_NAME}_gencfg)
 
 add_library(hector_gazebo_ros_sonar src/gazebo_ros_sonar.cpp)
 target_link_libraries(hector_gazebo_ros_sonar ${GAZEBO_LIBRARIES} ${catkin_LIBRARIES})

